src/status_im/contexts/wallet/home/view.cljs

```diff
@@ -170,6 +175,15 @@
      [tokens-loading?])
     [rn/view {:style (style/home-container)}
      [common.top-nav/view]
+     (when privacy-mode-enabled?
+       [quo/information-box
+        {:type                :informative
+         :customization-color :purple
+         :icon                :i/info
+         :on-button-press     show-privacy-mode-sheet
+         :button-label        (i18n/label :t/disable-privacy-mode)
+         :style               {:margin-vertical 8 :margin-horizontal 20}}
+        (i18n/label :t/features-and-balances-disabled)])
```

Comment:
Now seeing in the UI, it's a bit odd to read "Features and balances are disabled" because many features are not disabled, which makes me think as a user: "What does Status mean actually by features are disabled? Which ones are they talking about?"

Reply:
@Parveshdhull per our discussion in [Discord](https://discord.com/channels/1210237582470807632/1354212039123271823/1364397228474634300) the new text should be `Balances and some features are unavailable`.

Reply:
Thank you @ilmotta, updated.